.gitignore

```diff
@@ -9,4 +9,5 @@ modules/choose.py
 .pylintrc
 *.ipynb
 *.csv
-.vscode
\ No newline at end of file
+.vscode
+help.md
```

Comment:
it's the file generated from docstrings with `build_help.py`
you [suggested](https://github.com/StampyAI/stampy/pull/263#discussion_r1209477740) to keep only one copy of documentation in the repo

Reply:
I didn't notice there is any build help and thought you copy&pasted the info and wanted to have only 1 source file with documentation 😅 

in this case, why do we build the `help.md` file, how do we use it?

Reply:
It's just a markdown file that concatenates and nicely formats the documentation from all files which have a file-level docstring. The intention is to eventually have a single file with all Stampy commands. Also, this will be helpful for  #107